src/components/atoms/Icon/icons.ts

```diff
@@ -274,4 +275,5 @@ export default {
   compass: Compass,
   compassFocus: CompassFocus,
   house: House,
+  navigator: WidgetNavigator,
```

Comment:
Can you move this in to the same order as the imports? Just makes things a little easier to find

Reply:
I fixed this in [refactor: move navigator icon](https://github.com/reearth/reearth-web/pull/323/commits/d3457c6bd4c5671228f01eb2c5bb64a287e41b5d).